lambda_src/response_storage.py

```diff
@@ -0,0 +1,47 @@
+import os
+from typing import Dict, Text
+
+import boto3
+
+AWS_REGION = os.environ[
+    'AWS_REGION'
+]
+DYNAMODB_TABLE = 'geff-requests'
+DYNAMODB_RESOURCE = boto3.resource('dynamodb', region_name=AWS_REGION)
+table = DYNAMODB_RESOURCE.Table(DYNAMODB_TABLE)
+
+
+def write_dynamodb_item(batch_id: Text, response: Dict):
+    """
+    Write to the requests table, a batch id and a response
+    """
+    table.put_item(Item={'batch_id': batch_id, 'response': response})
+
+
+def initialize_dynamodb_item(batch_id: Text):
+    """
+    Initialize an item in the requests table with a null response
+    """
+    table.put_item(Item={'batch_id': batch_id, 'response': None})
+
+
+def get_response(batch_id: Text):
+    """
+    Retreive response for a batch id
+    """
+    item = table.get_item(Key={'batch_id': batch_id})
+    response = item['Item']['response']
+    return response
+
+
+def check_if_initialized(batch_id: Text):
+    """
+    Check if a response for a batch id exists in the table
+    """
+    item = table.get_item(Key={'batch_id': batch_id})
+    try:
+        response = item['Item']['response']
```

Comment:
when you initialize, the response is set to `None`. are you sure it will not return `None` here? add a comment explaining this, if so.

Reply:
Actually, I'd considered that it was fine when it did too, because the I wanted to mainly check for the existence of the batch_id key.

Reply:
and if the key existed, we'd get `True` returned

Reply:
what does `get_item` return when the key is not found? is there a better way to check?

Reply:
It returns a `None`. 

The other way is to use chained `get`s:

```
item.get("Item", {}).get("response")
```